js/ui/viewSelector.js

```diff
@@ -199,9 +314,99 @@ const ViewsDisplay = new Lang.Class({
     Name: 'ViewsDisplay',
 
     _init: function() {
+        this._enterSearchTimeoutId = 0;
+        this._localSearchMetricTimeoutId = 0;
```

Comment:
Unless I'm confused, it looks like this same line is added on two commits. End result is fine, but perhaps the commit history is confusing.